archiver/helpers.py

```diff
@@ -338,36 +339,36 @@ def file_is_valid_archive_or_terminate(file_path):
         terminate_with_message(f"File {file_path.as_posix()} is not a valid archive of type {COMPRESSED_ARCHIVE_SUFFIX} or {ENCRYPTED_ARCHIVE_SUFFIX} or doesn't exist.")
 
 
-def filename_without_extensions(path):
-    """Removes every suffix, including .partX"""
-    suffixes_string = "".join(path.suffixes)
-
-    return path.name[:-len(suffixes_string)]
+def filename_without_archive_extensions(path):
+    """Removes every archiving suffix"""
+    return filepath_without_archive_extensions(path).name
```

Comment:
I would put this function below `def filepath_without_archive_extensions` to make sure that this function always knows what exactly to call.